apps/builder/app/canvas/features/text-editor/toolbar-connector.tsx

```diff
@@ -275,3 +275,32 @@ export const ToolbarConnectorPlugin = ({
 
   return null;
 };
+
+export const ToolbarConnectorPlugin = ({
```

Comment:
ideally we get rid step by step things on canvas that rely on react, not add more of them